content/guides/identity-management/quick-start/2-configure-okta.md

```diff
@@ -0,0 +1,262 @@
+Connect integration allows us to use baked in OIDC connectors within a number
+of popular language frameworks to simplify application and user management by
+handling the callback routes and providing methods for logging in,
+logging out, and protecting routes into your application.
+
+To simplify this first integration, we're going to use the default callback
+routes and settings for the language and framework OIDC connector. Depending on
+your preferred integration type, the configuration settings will slightly
+change.
+
+Fill out the application details with the following configuration settings:
+
+<Choice option='programming.platform' value='node' color='none'>
+
+* Name: Any
+* Base URIs: `http://localhost:3000/`
+* Login redirect URIs: `http://localhost:3000/authorization-code/callback`
+* Logout redirect URIs: `http://localhost:3000/logout`
+* Grant type allowed: Only **Authorization Code** selected
+
+<ImageFrame noborder center shadow>
+  ![Okta App Configuration](./img/okta-qs-step2-appconfig-node.png)
+</ImageFrame>
+
+</Choice>
+<Choice option='programming.platform' value='java' color='none'>
+
+* Name: Any
+* Base URIs: `http://localhost:8080/`
+* Login redirect URIs: `http://localhost:8080/authorization-code/callback`
+* Logout redirect URIs: `http://localhost:8080/logout`
+* Grant type allowed: Only **Authorization Code** selected
+
+<ImageFrame noborder center shadow>
+  ![Okta App Configuration](./img/okta-qs-step2-appconfig-java.png)
+</ImageFrame>
+
+</Choice>
+<Choice option='programming.platform' value='python' color='none'>
+
+* Name: Any
+* Base URIs: `http://127.0.0.1:5000/`
+* Login redirect URIs: `http://127.0.0.1:5000/oidc/callback`
+* Logout redirect URIs: `http://127.0.0.1:5000/logout`
+* Grant type allowed: Only **Authorization Code** selected
+
+<ImageFrame noborder center shadow>
+  ![Okta App Configuration](./img/okta-qs-step2-appconfig-python.png)
+</ImageFrame>
+
+</Choice>
+<Choice option='programming.platform' value='cs' color='none'>
+
+* Name: Any
+* Base URIs: `https://localhost:5001/`
+* Login redirect URIs: `https://localhost:5001/authorization-code/callback`
+* Logout redirect URIs: `https://localhost:5001/logout`
+* Grant type allowed: Only **Authorization Code** selected
+
+<ImageFrame noborder center shadow>
+  ![Okta App Configuration](./img/okta-qs-step2-appconfig-cs.png)
+</ImageFrame>
+</Choice>
+
+<Choice option='programming.platform' unset color='none'>
+  <Message danger>
+    # Incomplete previous step
+    Please select a preferred language / framework in step 1 to get started.
+  </Message>
+</Choice>
+
+Click the **Done** button to create the application and be dropped on to the
+general settings of the application.
+
+## Copy Application Credentials 
+
+Using the configuration files set up in the last step, we next have to add in
+the Okta application org and app details within the files.
+
+Most Okta application information can be found on the general settings page,
```

Comment:
Maybe have people copy the client ID and secret first, and THEN direct them to the place to find the org URL? Also maybe add a code sample to say "this is what your config now looks like"?

Reply:
Added sections for how the config files should look. I think I'll keep the guidance for client / secret / domain as is for now. Although folks might follow the general guidance for the domain URL then go back into the app I think it helps reinforce where everything is as is.